.travis.yml

```diff
@@ -67,13 +67,13 @@ script:
   - |
     if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then
       if [[ "$BUILD_TARGET" == "qemu" && "$BUILD_ARCH" == "" ]]; then
-        docker run --rm -v ${PWD}:/tmp/sbl -w /tmp/sbl --network=host sbl bash -c "python3 ./BuildLoader.py build ${BUILD_REL} ${BUILD_TARGET} && python3 ./Platform/QemuBoardPkg/Script/qemu_test.py";
+        docker run --rm -v ${PWD}:/tmp/sbl -w /tmp/sbl --network=host sbl bash -c "python3 BootloaderCorePkg/Tools/GenerateKeys.py -k ./../SlimbootKeyDir && python3 ./BuildLoader.py build ${BUILD_REL} ${BUILD_TARGET} && python3 ./Platform/QemuBoardPkg/Script/qemu_test.py";

```

Comment:
./../  should be ../

Reply:
Do you recommend SlimBootKeyDir to be internal to SblOpen in our travis builds?

Reply:
./../ is the same as ../  
So ./ is not required.
+-SblKeys
 |
+-SblOpen 
They should be at the same level.

Reply:
Thanks Maurice.
 I have mis interpreted earlier response for  ../